x/auctions/keeper/grpc_query.go

```diff
@@ -0,0 +1,45 @@
+package keeper
+
+import (
+	"context"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/evmos/v18/utils"
+	"github.com/evmos/evmos/v18/x/auctions/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// AuctionInfo returns the current auction information
+func (k Keeper) AuctionInfo(c context.Context, _ *types.QueryCurrentAuctionInfoRequest) (*types.QueryCurrentAuctionInfoResponse, error) {
```

Comment:
I believe here could be useful to return the remaining time before the next bid.

Reply:
Hmm I was thinking about it but that will also require us to add the epoch info to the auctions module store which is unnecessary IMO. Better to keep this as a query from the client to display on the auctions page.